tests/functional/api/test_import_export.py

```diff
@@ -78,3 +78,23 @@ def test_project_remote_import(gl):
         "File url is blocked: Only allowed schemes are https"
         in err_info.value.error_message
     )
+
+
+def test_project_remote_import_s3(gl):
+    gl.features.set("import_project_from_remote_file_s3", True)
+    with pytest.raises(gitlab.exceptions.GitlabHttpError) as err_info:
+        gl.projects.remote_import_s3(
+            "aws-region",
+            "aws-bucket-name",
+            "aws-file-key",
+            "aws-access-key-id",
+            "secret-access-key",
+            "remote-project",
+            "remote-project",
+            "root",
+        )
+    assert err_info.value.response_code == 400
+    assert (
+        "Failed to open 'aws-file-key' in 'aws-bucket-name'"
+        in err_info.value.error_message
+    )
```

Comment:
To answer your question from the previous PR, I think `match` works like inclusion tests for plain strings so no need for regex. But actually this looks fine :)